NewRelicBrowser.php

```diff
@@ -1,449 +1,562 @@
+		}
+
+		/**
+		 * Register relic options settings.
+		 *
+		 * @return void
+		 */
+		public function rtp_relic_register_settings() {
 			register_setting( 'relic_options_settings', 'relic_options', array( $this, 'rtp_relic_options_validate' ) );
 		}
 
 		/**
 		 * Add options page
-		 * @return null
+		 *
+		 * @return void
 		 */
-		function rtp_relic_option_page()
-		{
-			$rtp_new_relic_setting_page = add_options_page( 'New Relic Options', 'New Relic Browser', 'manage_options', 'new-relic-browser', array( $this, 'new_relic_options' ) );
+		public function rtp_relic_option_page() {
+			$rtp_new_relic_setting_page = add_options_page( esc_html__( 'New Relic Options', 'rt-new-relic' ), esc_html__( 'New Relic Browser', 'rt-new-relic' ), 'manage_options', 'new-relic-browser', array( $this, 'new_relic_options' ) );
 			add_action( 'load-' . $rtp_new_relic_setting_page, array( $this, 'rtp_relic_page_help' ) );
 		}
 
 		/**
 		 * Add help tab
-		 * @return null
+		 *
+		 * @return void
 		 */
-		function rtp_relic_page_help()
-		{
+		public function rtp_relic_page_help() {
 			$screen = get_current_screen();
 			$screen->add_help_tab(
 				array(
-						'id' => 'rtp_relic_page_overview_tab',
-						'title' => __( 'Overview', 'rt-new-relic' ),
-						'content' => '<p>' . __(
-							'This page will allow you to integrate your New Relic Browser app with your website. 
-					If you do not have New Relic account, then just select "No" and provide required details. The New Relic script will be loaded automatically in &lt;head&gt; tag of your site without any manual effort.', 'rt-new-relic'
-						) . '</p>', )
+					'id'      => 'rtp_relic_page_overview_tab',
+					'title'   => esc_html__( 'Overview', 'rt-new-relic' ),
+					'content' => '<p>' . esc_html__(
+						'This page will allow you to integrate your New Relic Browser app with your website. 
+							If you do not have New Relic account, then just select "No" and provide required details. The New Relic script will be loaded automatically in &lt;head&gt; tag of your site without any manual effort.',
+						'rt-new-relic'
+					) . '</p>',
+				)
 			);
 			$screen->add_help_tab(
 				array(
-						'id' => 'rtp_relic_page_about_tab',
-						'title' => __( 'New Relic Browser', 'rt-new-relic' ),
-						'content' => '<p>' . __( 'New Relic Browser provides deep visibility and actionable insights into real users experiences on your website. With standard page load timing (sometimes referred to as real user monitoring or RUM), New Relic measures the overall time to load the entire webpage. However, New Relic Browser goes beyond RUM to also help you monitor the performance of individual sessions, AJAX requests, and JavaScript errors—extending the monitoring throughout the entire life cycle of the page.', 'rt-new-relic' ) . '</p>',
-					)
+					'id'      => 'rtp_relic_page_about_tab',
+					'title'   => esc_html__( 'New Relic Browser', 'rt-new-relic' ),
+					'content' => '<p>' . esc_html__(
```

Comment:
Use sprintf instead of concating long strings

Reply:
i solved this issue. please review changes.